### DISCUSSION TITLE
Disparity depth problems

### DISCUSSION THREAD

**Discussion by PeterParkerLogic** [microsoft/AirSim#3632] [Support Q&A]
1.How to get deNormalized disparity ? I do not understand what (Xl - Xr)/Xmax is. What are Xl ,Xr and Xmax?
2.What is the focal length of camera in airsim?

Thank you very much

**Comment by KailinTong** [comment]
Hi, 

I am facing a similar problem.
do you have any updates about this question?

Thanks a lot!

**Comment by PeterParkerLogic** [comment]
No, I still stuck on this problem. And I do not know how to solve this. Sorry

**Comment by jonyMarino** [comment]
Hi. Regarding 2 I think this PR can be useful: https://github.com/microsoft/AirSim/pull/3949
